### PR TITLE
Support new import / export syntax in preProcessFile

### PR DESCRIPTION
**Pull request by mhegazy**

### CODE REVIEW

File: src/services/services.ts [hunk 1]
```diff
@@ -1752,36 +1752,145 @@ module ts {
...
+                            else {
+                                // unknown syntax
+                                continue;
+                            }
+                        }
+
+                        if (token === SyntaxKind.OpenBraceToken) {
+                            token = scanner.scan();
+                            // consume "{ a as B, c, d as D}" clauses
+                            while (token !== SyntaxKind.OpenBraceToken && token !== SyntaxKind.CloseBraceToken) {
```
Comment (vladima) [inline src/services/services.ts]: why not just `while (token !== SyntaxKind.CloseBraceToken)`? when we might expect another OpenBraceToken?

Comment (mhegazy) [inline src/services/services.ts]: i was thinking of an unterminated brace, and did not want to eat up the whole file.. thought it does not make much difference..

File: src/services/services.ts [hunk 2]
```diff
@@ -1752,36 +1752,145 @@ module ts {
             });
         }
 
+
```
Comment (yuit) [inline src/services/services.ts]: extra space

File: src/services/services.ts (hunk 1 above)
Comment (yuit) [inline src/services/services.ts]: If you gonna leave the ```!== SyntaxKind.OpenBraceToken``` could you make comment of why you do that? I have similar question as @vladima

**Comment by vladima** [comment]
:+1:

**Pull request merged**